pandas/_libs/tslibs/tzconversion.pyi

```diff
@@ -0,0 +1,25 @@
+from datetime import (
+    timedelta,
+    tzinfo,
+)
+from typing import (
+    Iterable,
```

Comment:
from https://github.com/python/typeshed/blob/master/CONTRIBUTING.md#stub-file-coding-style

> in Python 3 stubs, import collections (Mapping, Iterable, etc.) from collections.abc instead of typing;